docs/changehistory/NextVersion.md

```diff
@@ -1,4 +1,107 @@
 ---
 publish: false
 ---
-# NextVersion
\ No newline at end of file
+# NextVersion
+
+## Simplified material creation
```

Comment:
@nick4598 haven't most of these changes been deleted on master?

Reply:
NextVersion.md is empty on master, but had content in imodel02 branch. Should NextVerson.md be empty?

Reply:
"Optimization of geometry in IModelImporter" section should be included - that was added on imodel02. I think you'll have to do some comparisons to find out what else was added on imodel02 and not present on master.

Reply:
Yep, you're right. Thanks for the heads up, looks like we may have missed a merge back into imodel02 during the last addon. I'll look into it.

Reply:
All other parts of nextversion.md were already present on master. I've updated nextversion.md